marketdata/stream/message_test.go

```diff
@@ -165,6 +165,25 @@ type cryptoBarWithT struct {
 	NewField uint64 `msgpack:"new"`
 }
 
+// cryptoOrderbookWithT is the incoming crypto orderbook message that also contains the T type key
+type cryptoOrderbookWithT struct {
+	Type      string                      `msgpack:"T"`
+	Symbol    string                      `msgpack:"S"`
+	Exchange  string                      `msgpack:"x"`
+	Timestamp time.Time                   `msgpack:"t"`
+	Bids      []cryptoOrderbookEntryWithT `msgpack:"b"`
+	Asks      []cryptoOrderbookEntryWithT `msgpack:"a"`
+	// NewField is for testing correct handling of added fields in the future
+	NewField uint64 `msgpack:"n"`
+}
+
+// cryptoOrderbookEntryWithT is the incoming crypto orderbook entry message that also contains the T type key
+type cryptoOrderbookEntryWithT struct {
```

Comment:
Should `cryptoOrderbookEntryWithT` has a T type key? I think it is unnecessary.

Reply:
You're right. Removed 👍